indico/modules/events/templates/display/_common.html

```diff
@@ -0,0 +1,104 @@
+{% macro _render_location_tip(item) -%}
+    {% set show_url = item.room and item.room.map_url %}
+    <div class="location-info-tip">
+        {% if item.room_name -%}
+            <h3>{{ item.room_name }}</h3>
+        {% endif %}
+        {% if item.venue_name -%}
+            <h4>{{ item.venue_name }}</h4>
+        {% endif %}
+        {% if item.address %}
+            <div class="location-address">
+                {#- -#}
+                {{ item.address }}
+            </div>
+        {%- endif %}
+        {% if item.room and item.room.capacity %}
+            <span class="icon-user">
+                {{ item.room.capacity }}
+            </span>
+        {% endif %}
+        {% if show_url %}
+            <a href="{{ item.room.map_url }}" class="map-link">
+                {% trans %}Show Room on Map{% endtrans %}
+            </a>
+        {% endif %}
+    </div>
+{%- endmacro %}
+
+{% macro render_location(item, parent=none, class='item-location') -%}
+    {% set show_venue = not parent or (parent.venue_name != item.venue_name) %}
+    {% set show_room = item.room_name != '' %}
+    {% if show_venue or show_room %}
+        <span class="icon-location location-info {{ class }}"
+              data-qtip-html=".qtip-hidden-content"
+              data-qtip-opts='{"hide": {"fixed": true, "delay": 1000}}'>
+            <span class="text">
+                {%- if show_venue and show_room -%}
+                    {{ item.room_name }} ({{ item.venue_name }})
+                {%- elif show_venue -%}
+                    {{ item.venue_name }}
+                {%- else -%}
+                    {{ item.room_name }}
+                {%- endif -%}
+            </span>
+        </span>
+        <span class="qtip-hidden-content">
+            {{ _render_location_tip(item) }}
+        </span>
+    {% endif %}
+{%- endmacro %}
+
+{% macro render_users(user_list, span_class='', title=true, italic_affiliation=false, separator=', ', use_span=true) -%}
+    {% for link in user_list -%}
+        {%- if use_span -%}
+            <span itemprop="performers" itemscope itemtype="http://schema.org/Person" class="{{ span_class }}">
+                {% if link.title -%}
+                    {{ link.title }}
+                {%- endif %}
+                <span itemprop="givenName">{{ link.first_name }}</span>
+                <span itemprop="lastName">{{ link.last_name }}</span>
+        {%- else -%}
+            {{ link.first_name }} {{ link.last_name }}
+        {% endif %}
+        {% if link.affiliation %}
+            <span class="affiliation">(
```

Comment:
I'd move the () inside the if/else. seems to be much clearer than having it in the same line as a tag